app/models/user/social_actions.rb

```diff
@@ -49,6 +49,11 @@ def build_message(conversation, opts={})
   end
 
   def find_or_create_participation!(target)
-    participations.where(:target_id => target).first || participate!(target)
+    participation = participations.where(:target_id => target).first
+    if participation.present?
+      participation.update!(count: participation.count.next)
+    else
+      participation = participate!(target)
```

Comment:
Useless assignment to variable - `participation`. Did you mean `participations`?